analyzers/src/SonarAnalyzer.CSharp/Rules/GenericTypeParametersRequired.cs

```diff
@@ -31,35 +31,50 @@ public sealed class GenericTypeParametersRequired : SonarDiagnosticAnalyzer
 
         protected override void Initialize(SonarAnalysisContext context) =>
             context.RegisterSyntaxNodeActionInNonGenerated(c =>
+            {
```

Comment:
note: it's better to split whitespace and non-functional refactoring in a separate commit, to ease reviewing of the PR

Reply:
Ooops. I may have unintentionally pressed Ctrl+K+D.
Reverted it.